src/renderers/contentful/renderAllLocales.ts

```diff
@@ -1,6 +1,10 @@
 import renderUnion from "../typescript/renderUnion"
 import { Locale } from "contentful"
+import { escapeSingleQuotes } from "../utils"
 
 export default function renderAllLocales(locales: Locale[]): string {
-  return renderUnion("LOCALE_CODE", locales.map(locale => `'${locale.code}'`))
+  return renderUnion(
+    "LOCALE_CODE",
+    locales.map(locale => `'${escapeSingleQuotes(locale.code)}'`),
```

Comment:
Sorry, I missed this one too 😅 Do you mind reverting this file too?

Reply:
Cool got it, I found one more sys.id escape too and i removed the escape for that too.